start.go

```diff
@@ -165,7 +175,12 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 				os.Exit(-1)
 			}
 
-			args := []string{"--kernel", kernel, "--initrd", initrd}
+			args := []string{
+				"--kernel", kernel,
+				"--initrd", initrd,
+				"--default_cpus", fmt.Sprintf("%d", defaultCpu),
+				"--default_memory", fmt.Sprintf("%d", defaultMemory),
```

Comment:
could you directly use `context.GlobalInt("default_cpus")` and
`context.GlobalInt("default_memory")` here?